docs/cookbook/mocking_hard_dependencies.rst

```diff
@@ -0,0 +1,93 @@
+.. index::
+    single: Cookbook; Mocking Hard Dependencies
+
+Mocking Hard Dependencies
```

Comment:
Perhaps "new Keyword" in brackets for readers who might not quite get what a hard dependency is right away? They are out there...